embark-ui/src/components/DebugButton.js

```diff
@@ -12,8 +12,9 @@ class DebugButton extends React.Component {
   }
 
   isDebuggable() {
-    return this.props.forceDebuggable ||
-      (this.props.contracts && this.props.contracts.find(contract => contract.address === this.props.transaction.to));
+    return this.props.transaction.hash && 
+      (this.props.forceDebuggable ||
```

Comment:
This one don't act as a force anymore, IMO we should either make it a real force or change the naming

Reply:
It wouldn't have worked anyway without a tx hash, so it's more of a safety check to prevent errors. This allowed us to pass in an `undefined` tx hash and hide the button, regardless of `forceDebuggable`